scripts/collect-assets.js

```diff
@@ -2,42 +2,74 @@
 import { ensureDir } from "fs-extra/esm";
 import { resolve, join } from "path";
 import { cwdToProjectRoot } from "./_util.js";
-import { downloadAndExtractTarGz, downloadGitHubRelease } from "./_download.js";
+import {
+  downloadAndExtractTarGz,
+  downloadGitHubRelease,
+  downloadWorkflowArtifacts,
+} from "./_download.js";
+import dotenv from "dotenv";
+dotenv.config();
 
 cwdToProjectRoot();
 
 // Setup download target
 const DOWNLOAD_DIR = resolve("downloaded_assets");
 await ensureDir(DOWNLOAD_DIR);
 
-// Download Kai assets
-await downloadGitHubRelease({
-  targetDirectory: join(DOWNLOAD_DIR, "kai/"),
-  metaFile: join(DOWNLOAD_DIR, "kai", "collect.json"),
+try {
+  if (process.env.WORKFLOW && process.env.WORKFLOW !== "False") {
```

Comment:
I'd rather use cli switches than envvars here.  This can be updated in a followup PR

Reply:
do you have any example I can refer to?

Reply:
would it be like `npm run collect-assets workflow=true` ?